the-list-be/package.json

```
@@ -11,6 +11,8 @@
   "license": "GPLv3",
   "dependencies": {
     "@koa/router": "^12.0.0",
-    "koa": "^2.13.4"
+    "koa": "^2.13.4",
+    "koa-body": "^5.0.0",
+    "mongodb": "4.10"
```

Думаю, можно разрешить фикс и фич апдейты.
```suggestion
    "mongodb": "^4.10.0"
```